src/components/menu/menu.svelte

```diff
@@ -243,8 +242,9 @@
   :global .leo-menu-popup ::slotted(*),
   :global .leo-menu-popup > * {
     --leo-menu-item-margin: var(--leo-spacing-s);
-    --leo-menu-item-padding: var(--leo-spacing-xl);
-    --leo-menu-item-border-radius: var(--leo-spacing-m);
+    --leo-menu-item-padding-horizontal: var(--leo-spacing-xl);
+    --leo-menu-item-padding-vertical: var(--leo-spacing-m);
```

Comment:
Don't think we need to separate that into two separate variables:

```suggestion
    --leo-menu-item-padding: var(--leo-spacing-m) var(--leo-spacing-xl);
```

This variable is just so people using the component can tweak it :smile:

Reply:
@fallaciousreasoning I accepted your suggestions there but it broke for me

<img width="994" alt="image" src="https://github.com/user-attachments/assets/eb117f17-bc92-4c22-8d0e-c67ffeffe549">